app/src/main/java/org/oppia/app/topic/TopicTab.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.topic
+
+enum class TopicTab(val position: Int) {
```

Comment:
Please add a KDoc for this enum to provide context on what it represents.

Reply:
Resolved